iointel/src/agent_methods/tools/agno/zoom.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional
+from agno.tools.zoom import ZoomTools as AgnoZoomTools
+from .common import make_base, wrap_tool
+
+
+class Zoom(make_base(AgnoZoomTools)):
+    def _get_tool(self):
+        return self.Inner(
+            account_id=self.account_id_,
+            client_id=self.client_id_,
+            client_secret=self.client_secret_,
+        )
+
+    @wrap_tool("agno__zoom__get_access_token", AgnoZoomTools.get_access_token)
+    def get_access_token(self) -> str:
+        return self.get_access_token(self)
```

Comment:
```suggestion
        return self._tool.get_access_token()
```
also not sure if this needs to be exposed as a tool